lib/convert.js

```diff
@@ -28,13 +28,14 @@
         Q = require("q");
 
     // Constants
-    var DEFAULT_IMAGE_QUALITY           = 90,
+    var DEFAULT_IMAGE_QUALITY           = "90",
         BACKGROUND_COLOR_FOR_FLATTENING = "#fff";
 
     function _getConvertArgumentsForSettings(settings, binaryPaths) {
         var args    = [],
             format  = settings.format,
-            quality = settings.quality ? String(settings.quality) : null,
+            intQuality = Number.parseInt(settings.quality),
```

Comment:
Nit: the radix parameter should always be specified according to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt